content/blog/posts/osl-alumni-Alex-Polvi-sells-startup.rst

```diff
@@ -0,0 +1,12 @@
+OSL Alumni Alex Plovi sells CoreOS to RedHat
```

Comment:
Alumni is the plural form and should be Alumnus technically. Please update here and below.